Kernel/ACPI/ACPIStaticParser.cpp

```diff
@@ -87,6 +87,15 @@ namespace ACPI {
         m_facs = find_table("FACS");
     }
 
+    const FADTFlags::HardwareFeatures& StaticParser::hardware_features() const
+    {
+        return m_hardware_flags;
+    }
+    const FADTFlags::x86_Specific_Flags& StaticParser::x86_specific_flags() const
+    {
+        return m_x86_specific_flags;
+    }
+
```

Comment:
Since these are virtual overrides, please decorate them with `virtual` and `override`.

Reply:
Oops, I misread this, never mind!